app/models/user.rb

```diff
@@ -31,6 +32,7 @@ def update_with_cas(access_token)
     self.given_name = access_token.extra.givenname || access_token.uid # Harriet
     self.family_name = access_token.extra.sn || access_token.uid # Tubman
     self.display_name = access_token.extra.displayname || access_token.uid # "Harriet Tubman"
+    self.university_id = access_token.extra.universityid || access_token.universityid # "999999999"
```

Comment:
I would suggest to change this line to 

```
self.university_id = access_token.extra.universityid 
```

There is no `access_token.universityid`. The `||` condition in the other fields (`given_name`, `family_name`) was to use fallback values, but in this case there is no fallback value.